src/core/window/clipboard.js

```diff
@@ -94,7 +84,16 @@ export function copyTextToClipboard(win, text, successCallback, failCallback) {
     /*
       Try copying with deprecated method as a fallback support.
     */
-    deprecatedCopyTextToClipboard(win, text, successCallback, failCallback);
+    if (deprecatedCopyTextToClipboard(win, text)) {
+      /* Clipboard successfully set using deprecated API */
+      successCallback();
+    } else {
+      /* Clipboard write failed using deprecated API */
+      failCallback();
+    }
+  } else {
```

Comment:
A few comments to simplify this a bit:
1. Comments of the form `/** ... */` typically indicate JSDoc format doc comments, whereas `//`-style comments are more typically used in this repo for inline explanatory comments
2. Combining the checks here could be intuitive and reduces a layer of `if` nesting
3. Since the anonymous functions being passed to the `.then(...)` here do nothing but call the success/failure callbacks, those callbacks can be passed directly.

```javascript
export function copyTextToClipboard(win, text, successCallback, failCallback) {
  if (!!win.navigator?.clipboard) {
    win.navigator.clipboard.writeText(text).then(successCallback, failCallback);
  } else if (isDeprecatedCopyingToClipboardSupported(win.document) &&
             deprecatedCopyTextToClipboard(win, text)) {
    successCallback();  
  } else {
    failCallback();
  }
}
```

Reply:
> A few comments to simplify this a bit:
> 
> 1. Comments of the form `/** ... */` typically indicate JSDoc format doc comments, whereas `//`-style comments are more typically used in this repo for inline explanatory comments
> 2. Combining the checks here could be intuitive and reduces a layer of `if` nesting
> 3. Since the anonymous functions being passed to the `.then(...)` here do nothing but call the success/failure callbacks, those callbacks can be passed directly.
> 
> ```js
> export function copyTextToClipboard(win, text, successCallback, failCallback) {
>   if (!!win.navigator?.clipboard) {
>     win.navigator.clipboard.writeText(text).then(successCallback, failCallback);
>   } else if (isDeprecatedCopyingToClipboardSupported(win.document) &&
>              deprecatedCopyTextToClipboard(win, text)) {
>     successCallback();  
>   } else {
>     failCallback();
>   }
> }
> ```

Resolved!